2010-05-24	Magnus Granberg		<zorry@gentoo.org>

		* gcc/config.in	Add ENABLE_CRTBEGINTS and ENABLE_ESP

--- gcc/config.in	2009-04-21 11:08:08.000000000 +0200
+++ gcc/config.in	2009-05-12 00:10:08.000000000 +0200
@@ -46,6 +46,12 @@
 #endif
 
 
+/* Define to 1 to enable crtbeginTS.o. */
+#ifndef USED_FOR_TARGET
+#undef ENABLE_CRTBEGINTS
+#endif
+
+
 /* Define to 1 to specify that we are using the BID decimal floating point
    format instead of DPD */
 #ifndef USED_FOR_TARGET
@@ -65,6 +65,12 @@
 #endif
 
 
+/* Define to 1 to enable esp. */
+#ifndef USED_FOR_TARGET
+#undef ENABLE_ESP
+#endif
+
+
 /* Define to 1 to enable fixed-point arithmetic extension to C. */
 #ifndef USED_FOR_TARGET
 #undef ENABLE_FIXED_POINT
